arm-siterecovery/2015-11-10/swagger/service.json

```diff
@@ -0,0 +1,13707 @@
+      "get": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Gets the details of a vCenter.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "vCenter name of interest.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VCenter"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "VCenters"
+        ],
+        "summary": "Perform Add vCenter operation.",
+        "description": "TODO (avrai): Detailed Description.",
+        "operationId": "VCenters_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          },
+          {
+            "$ref": "#/parameters/ResourceType"
+          },
+          {
+            "$ref": "#/parameters/ResourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "fabricName",
+            "in": "path",
+            "description": "The name of the fabric containing the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vCenterName",
+            "in": "path",
+            "description": "The name of the vCenter.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "addVCenterRequest",
+            "in": "body",
```

Comment:
SiteRecovery's REST API doc which was approved by ARM for reference:
https://microsoft.sharepoint.com/teams/cdm-idc/_layouts/15/WopiFrame.aspx?sourcedoc={C9F91552-0255-44B5-A57D-106275A37D14}&file=Azure%20Site%20Recovery%20CSMV2%20API%20Spec.docx&action=default